dev/io.openliberty.microprofile.openapi.4.0.internal/src/io/openliberty/microprofile/openapi40/internal/impl/package-info.java

```diff
@@ -0,0 +1,14 @@
+/*******************************************************************************
+ * Copyright (c) 2024, 2025 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+@com.ibm.websphere.ras.annotation.TraceOptions(traceGroup = Constants.TRACE_GROUP, messageBundle = Constants.TRACE_OPENAPI)
+@org.osgi.annotation.versioning.Version("0.1")
```

Comment:
```suggestion
@org.osgi.annotation.versioning.Version("1.0")
```
First version of an exported package is usually 1.0